src/services/anchor/index.ts

```diff
@@ -185,19 +198,35 @@ export async function sep12First(sessionParams: IAnchorSessionParams): Promise<v
   //>????
 }*/
 
-export async function sep24First(sessionParams: IAnchorSessionParams): Promise<ISep24Intermediate> {
+export async function sep24First(
+  sessionParams: IAnchorSessionParams,
+  sep10Account: string,
+  outputToken: OutputTokenType,
+): Promise<ISep24Intermediate> {
   if (config.test.mockSep24) {
     return { url: 'https://www.example.com', id: '1234' };
   }
 
   const { token, tomlValues } = sessionParams;
   const { sep24Url } = tomlValues;
 
-  // at this stage, assetCode should be defined, if the config is consistent.
-  const sep24Params = new URLSearchParams({
-    asset_code: sessionParams.tokenConfig.stellarAsset.code.string,
-    amount: sessionParams.offrampAmount,
-  });
+  let sep24Params;
+  if (outputToken == 'ars') {
```

Comment:
I would again avoid token specific hard coded logic here, but I understand that this is really specific logic that is only valid for Anclap and we might remove it again once they implement this compliantly (`account` is the SEP-24 instead of the SEP-10 account).

Reply:
Yes, me too. But Anclap, therefore `ars`, seems to be the only one requiring the `account` on the `sep-24`. It seems from our current [conversations](https://satoshipay.slack.com/archives/C07PECGDX4Y/p1731032156696839?thread_ts=1730990813.226219&cid=C07PECGDX4Y) that it will continue to be this way.